src/Tooltip/Tooltip.docs.json

```diff
@@ -9,6 +9,11 @@
       "name": "align",
       "type": "'left' | 'right'"
     },
+    {
+      "name": "aria-label",
+      "type": "string",
+      "description": "Should be utilised for label type tooltips and it is going to be used to label the tooltip trigger."
+    },
     {
       "name": "direction",
       "type": "'n' | 'ne' | 'e' | 'se' | 's' | 'sw' | 'w' | 'nw'",
```

Comment:
What do you think about adding the default value `"defaultValue": "n"` to the component documentation to make the default direction explicit?

Reply:
Ah great catch! Thank you 🙇🏻‍♀️